src/snark/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod constraints;
```

Comment:
We should re-export all of `ark-snark` here (for consistency with the other impls)